package.json

```diff
@@ -12,15 +12,32 @@
     "url": "https://github.com/eventualbuddha/keysim.js"
   },
   "scripts": {
-    "build": "gobble build dist -f",
-    "test": "npm run test:node && npm run test:browser",
-    "test:browser": "./node_modules/karma/bin/karma start karma.conf.js",
-    "test:node": "mocha -R spec --recursive test"
+    "build": "browserify . --standalone keysim > dist/keysim.js",
+    "test": "npm run build && npm run test:node && npm run test:node:dist && npm run test:browser",
```

Comment:
1. Tests the build
1. runs the test in node
1. runs the test in node with the built module
1. runs the test in browsers